spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/token/authentication/XsuaaJwtDecoder.java

```diff
@@ -24,18 +24,20 @@
 import com.sap.cloud.security.xsuaa.XsuaaServiceConfiguration;
 
 public class XsuaaJwtDecoder implements JwtDecoder {
-	private final Logger logger = LoggerFactory.getLogger(getClass());
+	protected final Logger logger = LoggerFactory.getLogger(getClass());
 
 	Cache<String, JwtDecoder> cache;
-	private String uaaDomain;
-	private OAuth2TokenValidator<Jwt> tokenValidators;
+	protected OAuth2TokenValidator<Jwt> tokenValidators;
+	protected PostValidationAction postValidationAction;
+	protected XsuaaServiceConfiguration xsuaaServiceConfiguration;
 
-	XsuaaJwtDecoder(XsuaaServiceConfiguration xsuaaServiceConfiguration, int cacheValidityInSeconds, int cacheSize,
-			OAuth2TokenValidator<Jwt> tokenValidators) {
+	protected XsuaaJwtDecoder(XsuaaServiceConfiguration xsuaaServiceConfiguration, int cacheValidityInSeconds, int cacheSize,
+			OAuth2TokenValidator<Jwt> tokenValidators, PostValidationAction postValidationAction) {
```

Comment:
please allow a list of PostValidationAction (s)

Reply:
ok